gateway-incoming.yaml

```diff
@@ -146,6 +154,36 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
+  /trigger-sync:
```

Comment:
Nit: It seems like this should probably be called `enqueue`, but that's just like, my opinion, man.

Reply:
Haha, well, I do want it convey info about what we're triggering or enqueueing, Dude.